packages/api/src/routes/internal.ts

```diff
@@ -267,4 +279,46 @@ router.post(
   })
 );
 
+/**
+ * GET /internal/cx-hie-status
+ *
+ * Retrieves the customer status of enabled HIEs via the Feature Flags.
+ *
+ * @param req.query.cxId - The cutomer's ID.
+ */
+router.get(
+  "/cx-hie-status",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const result = await getHieEnabledFFStatus(cxId);
+    return res.status(httpStatus.OK).json(result);
+  })
+);
+
+/**
+ * PUT /internal/cx-hie-status
+ *
+ * Updates the customer status of enabled HIEs via the Feature Flags.
+ *
+ * @param req.query.cxId - The cutomer's ID.
+ * @param req.query.cwEnabled - Whether to enabled CommonWell.
+ * @param req.query.cqEnabled - Whether to enabled CareQuality.
```

Comment:
We should prob also add Epic here. This is part of the onboarding (just usually later) and enablement of HIEs for a given cx.

Reply:
Done.